packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/snapshot_list.dart

```diff
@@ -125,12 +126,10 @@ class _SnapshotListTitle extends StatelessWidget {
             const SizedBox(width: denseRowSpacing),
           ],
           if (theItem is SnapshotDocItem)
-            Expanded(
-              child: Text(
-                'Snapshots',
-                overflow: TextOverflow.ellipsis,
-                style: textStyle,
-              ),
+            Icon(
+              Icons.help_outline,
+              size: questionMarkSize,
```

Comment:
Can we use `defaultIconSize` here to keep this consistent with other icon buttons throughout DevTools?